dashboard/templates/common.html

```diff
@@ -17,10 +17,7 @@
 
 {% block javascripts %}
 
-    <script src="https://cdn.plot.ly/plotly-1.10.2.min.js"></script>
-    <script src="https://code.jquery.com/jquery-2.2.3.js"></script>
-    <script src="{% static 'prototype/js/vendor/superagent.js' %}"></script>
-    <script src="{% static 'prototype/js/main.js' %}"></script>
+    <script src="{% static 'prototype/js/bundle.js' %}"></script>
```

Comment:
add your `bundle.js` in git, otherwise there is a lot of install/build work on the deployment.